carl/envs/rna/data/download_and_build_rfam_taneda.sh

```diff
@@ -0,0 +1,27 @@
+cd data/
+
+mkdir rfam_taneda
+cd rfam_taneda
+
+wget rna.eit.hirosaki-u.ac.jp/modena/v0028/linux/modena.dataset.tar.gz
+
+tar -xf modena.dataset.tar.gz
+
+rm -f modena.dataset.tar.gz
+rm -rf ct_version
+
+i=1
```

Comment:
What is happening here? Please add a comment

Reply:
I added some more comments, but what Theresa described is exactly what is happening